sw/airborne/peripherals/lis3mdl.c

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (C) 2019 Gautier Hattenberger <gautier.hattenberger@enac.fr>
+ *
+ * This file is part of paparazzi.
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @file peripherals/lis3mdl.c
+ *
+ * ST LIS3MDL 3-axis magnetometer driver interface (I2C).
+ */
+
+#include "peripherals/lis3mdl.h"
+
+#define LIS3MDL_ENABLE_AUTO_INC   (0x1<<7)
+
+#define LIS3MDL_STATUS_ZYXOR      0b10000000
```

Comment:
Shouldn't we move these registers to a separate file?

Reply:
that will make sense if we ever have two sensors with the same registers but for I don't think it is necessary